libs/api/domains/fiskistofa/src/lib/fiskistofa.resolver.ts

```diff
@@ -0,0 +1,97 @@
+import { FiskistofaClientService } from '@island.is/clients/fiskistofa'
+import { Args, Directive, Mutation, Query, Resolver } from '@nestjs/graphql'
+import { GetQuotaTypesForCalendarYearInput } from './dto/getQuotaTypesForCalendarYear.input'
+import { GetQuotaTypesForTimePeriodInput } from './dto/getQuotaTypesForTimePeriod.input'
+import { GetShipsInput } from './dto/getShips.input'
+import { GetShipStatusForCalendarYearInput } from './dto/getShipStatusForCalendarYear.input'
+import { GetShipStatusForTimePeriodInput } from './dto/getShipStatusForTimePeriod.input'
+import { GetSingleShipInput } from './dto/getSingleShip.input'
+import { UpdateShipQuotaStatusForTimePeriodInput } from './dto/updateShipQuotaStatusForTimePeriod.input'
+import { UpdateShipStatusForCalendarYearInput } from './dto/updateShipStatusForCalendarYear.input'
+import { UpdateShipStatusForTimePeriodInput } from './dto/updateShipStatusForTimePeriod.input'
+import { QuotaStatus } from './models/quotaStatus'
+import { QuotaType } from './models/quotaType'
+import { ShipBasicInfo } from './models/shipBasicInfo'
+import {
+  ExtendedShipStatusInformation,
+  ExtendedShipStatusInformationUpdate,
+  ShipStatusInformation,
+} from './models/shipStatusInformation'
+import { SingleShip } from './models/singleShip'
+
+const cacheTime = process.env.CACHE_TIME || 300
+const cacheControlDirective = (ms = cacheTime) => `@cacheControl(maxAge: ${ms})`
+
+@Resolver()
+export class FiskistofaResolver {
+  constructor(
+    private readonly fiskistofaClientService: FiskistofaClientService,
+  ) {}
+
+  @Directive(cacheControlDirective())
+  @Query(() => ExtendedShipStatusInformation)
```

Comment:
Please follow our [GraphQL naming conventions](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions) for all GraphQL queries, mutations, inputs and objects.

E.g. this could be `fiskistofaShipInformation(input: ...)` or `fiskistofaShip(shipNumber: ...) { catchQuotaCategories(timePeriod|calendarYear) { ... } }`

Reply:
Ok, sorry for not doing that. I'll change the names.